spec/models/map_spec.rb

```diff
@@ -553,6 +553,24 @@ def become_viewer(user)
 
         @map.can_add_layer(@user).should eq false
       end
+
+      it 'should take shared_empty_dataset into account' do
+        emptyDatasetName = Cartodb.config[:shared_empty_dataset_name]
```

Comment:
Minor, but ruby uses snake case (`empty_dataset_name`), not camel case.

You can use `rubocop` to check you styling. There's usually a plugin for your editor that can help. In Sublime Text, for example, the package is `SublimeLinter--rubocop`.

It will also tip you more efficient ways to write ruby code, so it's not just styling. I've learned a lot of things from it :laughing: